src/main/resources/static/api.yml

```diff
@@ -5211,16 +5243,13 @@ components:
               $ref: '#/components/schemas/TemporaryAccommodationApplication'
             allocatedToStaffMember:
               $ref: '#/components/schemas/TemporaryAccommodationUser'
+            status:
+              $ref: '#/components/schemas/TemporaryAccommodationAssessmentStatus'
           required:
             - application
     AssessmentSummary:
       type: object
       properties:
-        type:
```

Comment:
I think you need to keep this along with the discriminator for the frontend's OpenAPI tooling to work properly

Reply:
I've restored this property, but I had to remove the `enum` part of it to get it to compile.